zipline/testing/src/jsMain/kotlin/app/cash/zipline/testing/sealedClassSerializersJs.kt

```diff
@@ -31,14 +29,12 @@ class JsSealedClassMessageService : SealedClassMessageService {
     }
   }
 
-  override fun colorSwapFlow(request: FlowReference<SealedMessage>): FlowReference<SealedMessage> {
-    val requestFlow = request.take()
-    val flow: Flow<SealedMessage> = flow {
-      requestFlow.collect { message ->
+  override fun colorSwapFlow(flow: Flow<SealedMessage>): Flow<SealedMessage> {
+    return flow {
+      flow.collect { message ->
         emit(colorSwap(message))
       }
     }
```

Comment:
This is just a `flow.map(::colorSwap)` now, if you want.

Reply:
Ooooh yes please.